packages/ui/src/components/va-file-upload/VaFileUploadSingleItem/VaFileUploadSingleItem.vue

```diff
@@ -16,21 +19,30 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, PropType } from 'vue'
+import { defineComponent, ref, PropType } from 'vue'
+import { useStrictInject } from '../../../composables'
 
-import { VaButton } from '../../index'
+import { VaButton, VaFileUploadInject } from '../../index'
+import { VaFileUploadKey, ConvertedFile } from '../types'
 
-import { ConvertedFile } from '../types'
+const INJECTION_ERROR_MESSAGE = 'The VaFileUploadSingleItem component should be used in the context of VaFileUpload component'
 
 export default defineComponent({
   name: 'VaFileUploadSingleItem',
+
   components: {
     VaButton,
   },
+
   emits: ['remove'],
+
   props: {
     file: { type: Object as PropType<ConvertedFile | null>, default: null },
   },
+
+  setup: () => ({
+    disabled: useStrictInject<VaFileUploadInject>(VaFileUploadKey, INJECTION_ERROR_MESSAGE).disabled || false,
```

Comment:
in `VaFileUpload` we have this prop with default value:
`disabled: { type: Boolean, default: false },`
Why do we need a fallback value here?